pages/games/[id]/index.js

```diff
@@ -33,11 +36,50 @@ export default function DetailsPage({ isFavorite, toggleFavorite }) {
         width={400}
         height={210}
       ></StyledImageDisplay>
-      <StyledLink href="/">⬅️ Back</StyledLink>
-      <StyledDiv>
-        <StyledYearDisplay>{game.yearpublished}</StyledYearDisplay>
-        <StyledDesciption>{game.description}</StyledDesciption>
-      </StyledDiv>
+      <h1>{game.name}</h1>
+      <StyledTable>
+        <thead>
+          <tr>
+            <StyledTh>Rating</StyledTh>
+            <StyledTh>Age</StyledTh>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <StyledTd>{game.rating.slice(0, 3)}</StyledTd>
+            <StyledTd>{game.minAge}+</StyledTd>
+          </tr>
+        </tbody>
+        <thead>
+          <tr>
+            <StyledTh>Year Released</StyledTh>
+            <StyledTh>Playing Time</StyledTh>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <StyledTd>{game.yearpublished}</StyledTd>
+            <StyledTd>{game.playtime}</StyledTd>
+          </tr>
+        </tbody>
+        <thead>
+          <tr>           
+            <StyledTh>Min. Players</StyledTh>
+            <StyledTh>Max. Players</StyledTh>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>           
+            <StyledTd>{game.minPlayers}</StyledTd>
+            <StyledTd>{game.maxPlayers}</StyledTd>
+          </tr>
+        </tbody>
+      </StyledTable>
+      <StyledContents>Categories</StyledContents>
+      <StyledPTexts>{`${game.categories.slice(0, 7)}`}</StyledPTexts>
```

Comment:
```suggestion
      <StyledPTexts>{game.categories.slice(0, 7)}</StyledPTexts>
```
You don't need to create a template string if you just want to use a variable inside jsx.